src/chartLibraries/dygraph/index.js

```diff
@@ -267,6 +266,16 @@ export default (sdk, chart) => {
     }
   }
 
+  const makeColorOptions = () => {
+    const sparkline = chart.getAttribute("sparkline")
+    if (sparkline) return { colors: chart.getColors() }
+
+    const { dimensionIds } = chart.getPayload()
+    const colors = dimensionIds.map(id => chart.getDimensionColor(id))
+
+    return { colors }
+  }
+
```

Comment:
Just a question. Shouldn't `make` imply that this function returns a function?

Reply:
The convention on this file seems to be that `make` returns an object.